exchange/binance.go

```diff
@@ -3,6 +3,7 @@ package exchange
 import (
 	"context"
 	"fmt"
+	"github.com/rodrigo-brito/ninjabot/service"
```

Comment:
Please, organize the imports.
We can group standard libraries, external and internal modules

Reply:
done